controllers/API/comment.js

```diff
@@ -0,0 +1,97 @@
+var express = require('express');
+var router  = express.Router();
+var Comment = require('mongoose').model('comment');
+var Forum = require('mongoose').model('forum');
+
+
+router.post('/API/forum/comment',function (req, res) {
+    var newComment = new Comment({
+        text: req.body.text,
+        author: req.body.author,
+        posterID: req.body.posterID,
+        imgURL:req.body.imgURL
+    });
```

Comment:
You can also do `new Comment(req.body);` here.  A possible advantage or disadvantage (depending on how you look at it) is that Mongoose will then reject the new item if you provide nonsense parameters.

Reply:
That is indeed more neat I will have to look into how it behaves. As it is now mongoose already rejects nonsense parameters through the schema validation.